Client/Assets/MainFrameAtDocumentEnd.js

```diff
@@ -1 +1 @@
f("-"))<0&&(n=0),r=0;r<n;++r)t.charCodeAt(r)>=128&&d("not-basic"),m.push(t.charCodeAt(r));for(o=n>0?n+1:0;o<g;){for(i=v,a=1,s=36;o>=g&&d("invalid-input"),((l=(f=t.charCodeAt(o++))-48<10?f-22:f-65<26?f-65:f-97<26?f-97:36)>=36||l>p((u-v)/a))&&d("overflow"),v+=l*a,!(l<(c=s<=w?1:s>=w+26?26:s-w));s+=36)a>p(u/(h=36-c))&&d("overflow"),a*=h;w=x(v-i,e=m.length+1,0==i),p(v/e)>u-b&&d("overflow"),b+=p(v/e),v%=e,m.splice(v++,0,b)}return y(m)}function A(t){var e,n,r,o,i,a,s,l,c,h,f,g,v,y,_,A=[];for(g=(t=b(t)).length,e=128,n=0,i=72,a=0;a<g;++a)(f=t[a])<128&&A.push(m(f));for(r=o=A.length,o&&A.push("-");r<g;){for(s=u,a=0;a<g;++a)(f=t[a])>=e&&f<s&&(s=f);for(s-e>p((u-n)/(v=r+1))&&d("overflow"),n+=(s-e)*v,e=s,a=0;a<g;++a)if((f=t[a])<e&&++n>u&&d("overflow"),f==e){for(l=n,c=36;!(l<(h=c<=i?1:c>=i+26?26:c-i));c+=36)_=l-h,y=36-h,A.push(m(w(h+_%y,0))),l=p(_/y);A.push(m(w(l,0))),i=x(n,v,r==o),n=0,++r}++n,++e}return A.join("")}s={version:"1.3.2",ucs2:{decode:b,encode:y},decode:_,encode:A,toASCII:function(t){return v(t,(function(t){return c.test(t)?"xn--"+A(t):t}))},toUnicode:function(t){return v(t,(function(t){return l.test(t)?_(t.slice(4).toLowerCase()):t}))}},void 0===(o=function(){return s}.call(e,n,e,t))||(t.exports=o)}()}).call(this,n(0)(t),n(1))},function(t,e,n){"use strict";t.exports={isString:function(t){return"string"==typeof t},isObject:function(t){return"object"==typeof t&&null!==t},isNull:function(t){return null===t},isNullOrUndefined:function(t){return null==t}}},function(t,e,n){"use strict";e.decode=e.parse=n(29),e.encode=e.stringify=n(30)},function(t,e,n){"use strict";function r(t,e){return Object.prototype.hasOwnProperty.call(t,e)}t.exports=function(t,e,n,i){e=e||"&",n=n||"=";var a={};if("string"!=typeof t||0===t.length)return a;var s=/\+/g;t=t.split(e);var u=1e3;i&&"number"==typeof i.maxKeys&&(u=i.maxKeys);var l=t.length;u>0&&l>u&&(l=u);for(var c=0;c<l;++c){var h,f,p,m,d=t[c].replace(s,"%20"),g=d.indexOf(n);g>=0?(h=d.substr(0,g),f=d.substr(g+1)):(h=d,f=""),p=decodeURIComponent(h),m=decodeURIComponent(f),r(a,p)?o(a[p])?a[p].push(m):a[p]=[a[p],m]:a[p]=m}return a};var o=Array.isArray||function(t){return"[object Array]"===Object.prototype.toString.call(t)}},function(t,e,n){"use strict";var r=function(t){switch(typeof t){case"string":return t;case"boolean":return t?"true":"false";case"number":return isFinite(t)?t:"";default:return""}};t.exports=function(t,e,n,s){return e=e||"&",n=n||"=",null===t&&(t=void 0),"object"==typeof t?i(a(t),(function(a){var s=encodeURIComponent(r(a))+n;return o(t[a])?i(t[a],(function(t){return s+encodeURIComponent(r(t))})).join(e):s+encodeURIComponent(r(t[a]))})).join(e):s?encodeURIComponent(r(s))+n+encodeURIComponent(r(t)):""};var o=Array.isArray||function(t){return"[object Array]"===Object.prototype.toString.call(t)};function i(t,e){if(t.map)return t.map(e);for(var n=[],r=0;r<t.length;r++)n.push(e(t[r],r));return n}var a=Object.keys||function(t){var e=[];for(var n in t)Object.prototype.hasOwnProperty.call(t,n)&&e.push(n);return e}}]);
```

Comment:
I am actually unsure why this changed. My code should not have impacted this, but running bootstrap updated it, maybe a dep updated or something.